### DISCUSSION TITLE
Leech and Mirror Log

### DISCUSSION THREAD

**Discussion by amir02120** [anasty17/mirror-leech-telegram-bot#865] [General]
Hi, 
I was just wondering are u still planning to add the ability to log all the leeched files and mirrored files in a seperated channels as well?
I found some repoes do that but beleive me they suck at something else as soon as they try to personalize the repo for better results for themselves
Back then I asked u this in gp u had in TG u said planning to add it but seems not added yet.

:)

**Comment by anasty17** [comment]
I have removed all supergroup features. Just use the repo and try to teach other how to create their own bot instead of managing group. There is many groups now. And all of them create groups to show off, mental issues i guess.
2nd I didn't try others repo cz I can see how much the code bad in quality. Also it will be hard on them to pull the incoming features.  So I will not add any supergroup feature.

**Reply by amir02120** [reply]
Alright LOL but I was going to use it for personal use not to show off or being a manger of leech group.
Anyway I got it ok then no problem

**Reply by anasty17** [reply]
Then why leech log!
